app_feup/lib/model/entities/lecture.dart

```diff
@@ -109,11 +112,15 @@ class Lecture {
         lec.classNumber);
   }
 
+  // TODO: wtf does cloneHtml even mean?
```

Comment:
It looks like the `startTimeSeconds` attribute is only defined on lectures retrieved via the api endpoint, hence the need for the two factories. Don't quote me on that though, @andrefmrocha should give better input.